src/core/Akka.Remote/Transport/TestTransport.cs

```diff
@@ -64,27 +62,29 @@ public TestTransport(Address localAddress, AssociationRegistry registry, long ma
                 new SwitchableLoggedBehavior<bool, Tuple<Address, TaskCompletionSource<IAssociationEventListener>>>(
                     x => DefaultListen(), x => _registry.LogActivity(new ListenAttempt(LocalAddress)));
             AssociateBehavior =
-                new SwitchableLoggedBehavior<Address, AssociationHandle>(DefaultAssociate, address => registry.LogActivity(new AssociateAttempt(LocalAddress, address)));
-            ShutdownBehavior = new SwitchableLoggedBehavior<bool, bool>(x => DefaultShutdown(), x => registry.LogActivity(new ShutdownAttempt(LocalAddress)));
+                new SwitchableLoggedBehavior<Address, AssociationHandle>(DefaultAssociate,
+                    address => registry.LogActivity(new AssociateAttempt(LocalAddress, address)));
+            ShutdownBehavior = new SwitchableLoggedBehavior<bool, bool>(x => DefaultShutdown(),
+                x => registry.LogActivity(new ShutdownAttempt(LocalAddress)));
             DisassociateBehavior = new SwitchableLoggedBehavior<TestAssociationHandle, bool>(x =>
             {
                 DefaultDisassociate(x);
-                return Task.Run(() => true);
+                return Task.FromResult(true);
```

Comment:
I made small TPL changes in a number of places - mostly replacing no-op `Task.Run` calls with `Task.FromResult`.